src/test/regress/expected/multi_router_planner.out

```diff
@@ -1046,8 +1294,8 @@ PREPARE author_articles(int) as
 	FROM articles_hash
 	WHERE author_id = $1;
 EXECUTE author_articles(1);
-DEBUG:  Creating router plan
 DEBUG:  predicate pruning for shardId 840001
+DEBUG:  Creating router plan
```

Comment:
We could only know about router plannableness after shard pruning. Do you have a placement suggestion ? 

I think, we would end up removing this message in the long run.